Model/ResourceModel/CreditMemo.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * @category    ClassyLlama
+ * @copyright   Copyright (c) 2017 Classy Llama Studios, LLC
+ */
+
+namespace ClassyLlama\AvaTax\Model\ResourceModel;
+
+class CreditMemo extends \Magento\Framework\Model\ResourceModel\Db\AbstractDb
+{
```

Comment:
It appears that the database field names are used in a few places, and I would suggest setting a constant for each of the field names on the resource and reference those instead of having several strings typed out in the code directly representing the field names. This will allow for more strongly typed code, and the ability to find any references where those field names are used throughout the code. It also aids in any event that the field names need to be changed in the future as they are only typed out in one location.

You can find an example of this to this on  \ClassyLlama\AvaTax\Model\ResourceModel\Queue

```
    /**#@+
     * Field Names
     */
    const QUEUE_STATUS_FIELD_NAME = 'queue_status';
    const UPDATED_AT_FIELD_NAME = 'updated_at';
    const CREATED_AT_FIELD_NAME = 'created_at';
    /**#@-*/
```